### PR TITLE
Add Finance Tracker Website

### LINKED ISSUE #186: Add Finance Tracker Website
**Describe the project you want to add with tech stack**
I would like to propose adding a new project to the repository. The project is called "Finance Tracker Website." This web application allows users to manage their personal finances by tracking income and expenses. It includes features such as transaction management, categorization, transaction history, and a visually appealing user interface.

**Tech Stack:**
- Frontend: HTML, CSS, JavaScript
- Backend: Not applicable (front-end only)
- Libraries: None (vanilla JavaScript)
- Frameworks: None
- Database: None (data stored locally)
- Hosting: Not applicable (static website)

**Expected behavior**
I expect that this project proposal will be reviewed and, if approved, added to the repository as a new challenge. The challenge should include the project description, tech stack details, and any relevant information to help contributors understand and work on the project.

**Screenshots (optional)**
https://github.com/sujitmahapatra/frontend-mini-challenges/blob/0a36075c73786b8f1ebd360c11938ca416bf70a7/native/machine-coding/Finance%20Tracker/Finance%20Tracker.png

**Additional context (optional)**
This project is designed to help individuals manage their personal finances more effectively. It can serve as a useful tool for learning and practicing web development skills, especially in the areas of HTML, CSS, and JavaScript. It's intended to be a beginner-friendly challenge that contributors can use to enhance their web development skills.

### PR DESCRIPTION
**Pull request by sujitmahapatra**
# Title and Issue number

Title: Add Finance Tracker Website

Issue No.: #186

Code Stack: 
- Frontend: HTML, CSS, JavaScript

Close #186

# Checklist:

- [x] I have mentioned the issue number in my Pull Request.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have created a helpful and easy-to-understand `README.md`.
- [x] I have updated the `index.html` file for my contribution.

### CODE REVIEW

File: native/helpers/challenges.js [hunk 1]
```diff
@@ -7,6 +7,14 @@ export const challenges = [
     youtube: 'https://youtu.be/9OIeUtXX3SA',
     tags: [],
   },
+  {
+    title: 'Personal Finance Tracker',
+    link: 'native/machine-coding/Finance Tracker',
```
Comment (sadanandpai) [inline native/helpers/challenges.js:12]: this is not the valid link. plz refer to other challenges

File: native/machine-coding/Finance Tracker/README.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+3. Make your changes and commit them.
+4. Push your changes to your fork.
+5. Submit a pull request to the main repository.
+
+## Author
+
+- Your Name
+- GitHub: [Mr. Sujit](https://github.com/sujitmahapatra)
+
+Happy tracking your finances!
```
Comment (sadanandpai) [inline native/machine-coding/Finance Tracker/README.md:60]: Please add description about the challenge only

File: native/machine-coding/Finance Tracker/index.html [hunk 3]
```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Personal Finance Tracker</title>
+    <link rel="stylesheet" href="styles.css">
+    <script src="../../helpers/navbar.js" type="module"></script>
```
Comment (sadanandpai) [inline native/machine-coding/Finance Tracker/index.html:8]: add header.js only to the head (as per the new standards)
refer [counter](https://github.com/sadanandpai/frontend-mini-challenges/blob/main/javascript/machine-coding/counter/index.html)

File: native/machine-coding/Finance Tracker/styles.css [hunk 4]
```diff
@@ -0,0 +1,131 @@
+@import url(../../helpers/core.css);
```
Comment (sadanandpai) [inline native/machine-coding/Finance Tracker/styles.css:1]: can remove this

File: native/machine-coding/Finance Tracker/styles.css [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+body {
+    font-family: Arial, sans-serif;
+    margin: 0;
+    padding: 0;
```
Comment (sadanandpai) [inline native/machine-coding/Finance Tracker/styles.css:5]: font family, margin, padding are not needed

**Review by sadanandpai — CHANGES_REQUESTED** [review]

**Review by sujitmahapatra — COMMENTED** [review]
I've addressed the feedback and made the necessary changes.
I look forward to any final feedback.
Thank You!

**Review by sadanandpai — APPROVED** [review]

**Pull request merged**